From 754fc039e00a95e991cc4ea5f05c83d038a5f68f Mon Sep 17 00:00:00 2001
From: wwidjaja <w.widjaja@intel.com>
Date: Thu, 3 Sep 2020 15:52:21 +0800
Subject: [PATCH] PONRTSYS-4670 - remove UART MUX config

  do not set UART1 MUX / GPIO Portmux 30 & 31 that control
  UART1 output selection. up to uboot now to configure GPIO 30&31
  if someone wish to show TEP console.
---
 arch/mips/lantiq/grx500_bootcore/early_printk.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/arch/mips/lantiq/grx500_bootcore/early_printk.c b/arch/mips/lantiq/grx500_bootcore/early_printk.c
index 98ef696c93fe..233aee316b27 100644
--- a/arch/mips/lantiq/grx500_bootcore/early_printk.c
+++ b/arch/mips/lantiq/grx500_bootcore/early_printk.c
@@ -206,12 +206,9 @@ int __init setup_early_lntq_console(char *cmdline)
 	UART16550_WRITE(REG_CPU_SUB_CLK_GATE, CPU_SUB_CLK_GATE_MASK_ALL);	// CPU_SUB_CLK_GATE all gated
 	UART16550_WRITE(REG_CPU_SUB_RST, CPU_SUB_RST_MASK_RST);		// CPU_SUB_RST all reset
 	UART16550_WRITE(REG_UART_CONTROL, UART16550_TX_OUTPUT_DISABLE);	// CPU_UART_CTL TX_OE tx out disable
-	#ifdef CONFIG_SOC_PRX300_BOOTCORE
-	ltq_w32(LTQ_MUX_TEP, LTQ_PORTMUXC30); // GPI0 30 MUX -> TEP UART RX
-	ltq_w32(LTQ_MUX_TEP, LTQ_PORTMUXC31); // GPI0 31 MUX -> TEP UART TX
-	#elif defined(CONFIG_SOC_GRX500_BOOTCORE) /* CONFIG_SOC_PRX300_BOOTCORE */
+	#if defined(CONFIG_SOC_GRX500_BOOTCORE)
 	ltq_w32_mask(IFMUX_CFG_BBTEPUART_EN_MASK, LTQ_MUX_TEP, LTQ_IFMUX_CFG); // IFMUX_CFG , BBTEPUART_EN=TEP_U
-	#endif /* CONFIG_SOC_PRX300_BOOTCORE */
+	#endif /* CONFIG_SOC_GRX500_BOOTCORE */
 
 	//config HW
 	console_config();
